lib/tasks/export.rake

```diff
@@ -1,10 +1,10 @@
 namespace :export do
   namespace :csv do
     task :extracts, [:workflow_id] => [:environment] do |t, args|
-      Exporters::CsvExtractExporter.new.dump(args[:workflow_id])
+      Exporters::CsvExtractExporter.new(:workflow_id => args[:workflow_id]).dump
```

Comment:
As Hound would normally remark, prefer Ruby 1.9+ hash syntax (`.new(workflow_id: args[:workflow_id])`). The only time that doesn't work is if you don't want symbols as keys (for instance when constructing something that would come from parsing JSON, which always has strings as keys).